source/class/qx/tool/cli/commands/Package.js

```diff
@@ -131,8 +131,9 @@ qx.Class.define("qx.tool.cli.commands.Package", {
 
     /**
      * Convenience method to return all config file models as an array
-     * @return {Promise<[{qx.tool.config.Manifest}, {qx.tool.config.Lockfile}, {qx.tool.config.Compile}]>}
+     * @return {Promise<qx.tool.config.Manifest, qx.tool.config.Lockfile>} 
```

Comment:
should be `@return [] an array containing {qx.tool.config.Manifest} and {qx.tool.config.Lockfile}`